### ISSUE TITLE
[Bug]: Clock of Father Time and Eye of Chaos not craftable due to abyss change

### ISSUE THREAD

**Issue by gxam** [AllTheMods/Spellbound#73]
### Possible Fixes

Yes

### Modpack Version

other

### What happened?

ATMS 2.2.2.  The end_of_time_book from the abyss is gone in this version which breaks the recipe for father time.

```
[10Jan2022 14:01:29.654] [Render thread/WARN] [KubeJS Server/]: Error creating recipe astralsorcery:kjs_8950573zv730o85djxitbmvq7[astralsorcery:altar]: {"type":"astralsorcery:altar","altar_type":3,"duration":800,"starlight":6200,"pattern":["TTTTT","TCGHT"," TNT ","TBSOT","TTTTT"],"key":{"T":{"item":"twilightforest:time_wood"},"C":{"item":"create:cuckoo_clock"},"G":{"item":"ars_nouveau:glyph_extend_time"},"H":{"item":"botania:hourglass"},"N":{"item":"minecraft:clock"},"B":{"item":"theabyss:end_of_time_book"},"S":{"item":"supplementaries:clock_block"},"O":{"item":"tiab:timeinabottle"}},"output":[{"item":"kubejs:clock_of_father_time"}],"focus_constellation":"astralsorcery:horologium","relay_inputs":[{"tag":"forge:dusts/iesnium"},{"tag":"forge:dusts/iesnium"},{"tag":"forge:dusts/iesnium"}],"effects":["astralsorcery:built_in_effect_constellation_finish","astralsorcery:liquid_burst","astralsorcery:built_in_effect_trait_relay_highlight","astralsorcery:built_in_effect_discovery_central_beam","astralsorcery:built_in_effect_trait_focus_circle","astralsorcery:altar_default_sparkle","astralsorcery:built_in_effect_constellation_lines","astralsorcery:built_in_effect_attunement_sparkle"]}: com.google.gson.JsonSyntaxException: Unknown item 'theabyss:end_of_time_book'
```

### latest.log

https://paste.ee/p/zjJtX

**Comment by gxam** [comment]
Looks like the Eye of Chaos is also uncraftable.  The recipe shows up in JEI, but it doesn't seem to work.  Looking at the kubejs I see it is supposed to have theabyss:scorpioneye as an ingredient but that's not showing in JEI.

There is still a scorpion eye but it seems to be called theabyss:scoripon_eye now.

**Comment by Mitchell5200** [comment]
Confirmed fixed in V2.2.7, Closing
